cmlreaders/test/test_path_finder.py

```diff
@@ -18,7 +18,7 @@ def all_files_subject(rhino_root):
 @pytest.fixture
 def ramulator_files_finder(rhino_root):
     finder = PathFinder('R1354E', rootdir=rhino_root, experiment='PS4_FR5',
-                        session=0)
+                        session='0')
```

Comment:
We should really be using ints here and converting to strings as needed.

Reply:
Changing these as well.